### PR TITLE
Js classes

### PR DESCRIPTION
**Pull request by ZabihullahNooriWardak**
in this branch we worked on : 
1: added classes instead of objects.
2: changed js

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -12,23 +12,32 @@ if (arrayFromLocalStorage !== null) {
...
+    containerForSingleBook.classList.add('containerBooks');
+    containerForSingleBook.innerHTML = `<div class=bookDetails><p id='bookName'>${this.bookName}</p> <p id='by'>by</p> <p id='bookAuthor'>${this.bookAuthor}</p></div><button class="remove">remove</button>`;
+    containerForAllBooks.appendChild(containerForSingleBook);
+    const singleBookObject = new SingleBook(this.bookName, this.bookAuthor);
+    allBooks.push(singleBookObject);
+    localStorage.setItem('allBooks', JSON.stringify(allBooks));
+    bookTextField.value = '';
+    authorTextField.value = '';
+  }
+}
```
Comment (Strangeal) [inline script.js]: - [ ] Great job in refactoring the `add` functionality into a class method. However, Your `remove` and `display`  functionalities haven't been refactored into class methods as specified in the project requirements. To fulfill the project requirements for this milestone kindly consider refactoring both the `remove` and `display` functionalities into class methods like you have done for the `add` functionality.

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @ZabihullahNooriWardak and @Shahid-Bagwan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Well-structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi team @ZabihullahNooriWardak, @Shahid-Bagwan 🙋‍♀️,  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 
You have submitted the project using the wrong submission link. This link is for the `Awesome books: a complete website with navigation`, yet the milestone you are doing is `refactoring with ES6 classes`. Kindly request another review using the correct submission link.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @ZabihullahNooriWardak👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**